cmis_web_proxy/controllers/cmis.py

```diff
@@ -267,7 +266,8 @@ def _check_provided_token(self, cmis_path, proxy_info, params):
         if token:
             token = token.replace('Bearer', '').strip()
         else:
-            token = params.get('token').strip()
+            token = params.has_key('token') and params.get(
```

Comment:
@max3903  if params is a dict `token = params.get('token', '').strip()` otherwise `token = (params.get('token') or '').strip()`